projects/trafficserver/project.yaml

```diff
@@ -0,0 +1,14 @@
+homepage: "https://trafficserver.apache.org"
+language: c++
+primary_contact: "security@apache.org"
```

Comment:
Do you have permission from upstream? Do they know they wont be able to see any bugs with this?

Reply:
If you would like to send security alerts for the Apache Traffic Server project, you can send emails to security@trafficserver.apache.org